src/js/services/feeService.js

```diff
@@ -28,7 +28,7 @@ angular.module('copayApp.services').factory('feeService', function($log, bwcServ
       var feeLevelValue = lodash.find(levels, {
         level: feeLevel
       });
-      if (!feeLevelValue || !feeLevelValue.feePerKB)
+      if (!feeLevelValue || feeLevelValue.feePerKB == null)
```

Comment:
Normally i would agree, using `===` over `==` is usually safer however, my reason for doing it like this here is that `=== null` won't catch `undefined` .